docker-compose.yml

```diff
@@ -1,4 +1,11 @@
 version: '2'
+networks:
+  default:
```

Comment:
would you please find a better name, I do not think default is what you mean here

Reply:
Hi buddy that was an old branch I forgot to delete